packages/cli/src/commands/certs/command.ts

```diff
@@ -0,0 +1,116 @@
+import { Command } from '../help';
+import { packageName } from '../../util/pkg-name';
+
+export const certsCommand: Command = {
+  name: 'certs',
+  description:
+    'Interact with SSL certificates. This command is intended for advanced use only. By default, Vercel manages your certificates automatically.',
+  arguments: [
+    {
+      name: 'command',
+      required: false,
+    },
+  ],
+  subcommands: [
+    {
+      name: 'ls',
+      description: 'Show all available certificates',
+      arguments: [],
+      options: [],
+      examples: [],
+    },
+    {
+      name: 'issue',
+      description: ' Issue a new certificate for a domain',
+      arguments: [
+        {
+          name: 'cn',
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (soft-blocking):** ‏do we want to still show this accepts any number of domain arguments?

The example shows this. So does the old help output:

```
issue     <cn> [<cn>]
```

Side point: I don't know what `cn` is supposed to stand for. These are domains.